docs/portal_config.md

```diff
@@ -310,6 +310,7 @@ Below is an example, with inline comments describing what each JSON block config
         "rightIcon": "download"
       }
     ],
+    "loginForButtonFeature": true, //optional; redirects user to login page if they tries to download data without logging in.
```

Comment:
minor: I know we went back and forth on this, but since we are doing redirect for download buttons only, you should rename this to `loginForDownload` to be more clearly reflect what this is doing

Reply:
you also need to change here